20221125-132229
0.11298894882202148
898509
Case 1
=== in_idx ===
[ 1  2  4  6  7  8  9 10 12]
Traceback (most recent call last):
  File "prun_train/prune.py", line 256, in <module>
    main(exp,args)
  File "prun_train/prune.py", line 222, in main
    loose_model, prune_mask_dict, prune_bn_dict, mask_bn_channel_dict)
  File "prun_train/prune.py", line 142, in init_weights_from_loose_model
    w = loose_layer.weight.data[:, in_idx, :, :].clone()
IndexError: index 1 is out of bounds for dimension 0 with size 1
done
